tools/rc/usResourceCompiler.cpp

```diff
@@ -137,6 +135,17 @@ static FILE* us_fopen(const char* filename, const char* mode)
   return file;
 }
 
+std::string us_tempfile()
+{
+  char szTempFileName[MAX_PATH];
+  if (GetTempFileNameA(".", "ZIP", 1, szTempFileName) == 0)
+  {
+    char* err_str = get_error_str();
```

Comment:
```get_error_str``` will leak memory and while I understand that all instances of ```get_error_str``` result in the process terminating, I believe that a comment block above ```get_error_str``` explaining how to use the function would be useful for others modifying this code.